### ISSUE TITLE
Calculating level takes a long time

### ISSUE THREAD

**Issue by iRedFox** [BentoBoxWorld/Level#234]
### Expected behavior

It should calculate level when /is level is executed by player

### Observed/Actual behavior

It takes a long time and doesn't calculate blocks.
For example, after restart: it calculated 
(on join), but placed some blocks, did the /is level command

### Steps/models to reproduce

1- Place blocks
2- /is level
3- You will be stuck in Calculating level... Estimated wait: 2 seconds

### BentoBox version

[10:04:43 INFO]: Running PAPER 1.17.1.
[10:04:43 INFO]: BentoBox version: 1.17.2
[10:04:43 INFO]: Database: JSON
[10:04:43 INFO]: Loaded Game Worlds:
[10:04:43 INFO]: ASkyBlock (BSkyBlock): Overworld
[10:04:43 INFO]: Loaded Addons:
[10:04:43 INFO]: Biomes 1.14.0 (ENABLED)
[10:04:43 INFO]: Border 3.1.2-SNAPSHOT-b119 (ENABLED)
[10:04:43 INFO]: BSkyBlock 1.15.1 (ENABLED)
[10:04:43 INFO]: Challenges 0.9.0-SNAPSHOT-b457 (ENABLED)
[10:04:43 INFO]: ControlPanel 1.13.0 (ENABLED)
[10:04:43 INFO]: DimensionalTrees 1.6.0 (ENABLED)
[10:04:43 INFO]: IslandFly 1.8.1 (ENABLED)
[10:04:43 INFO]: Level 2.7.2 (ENABLED)
[10:04:43 INFO]: Limits 1.15.5 (ENABLED)
[10:04:43 INFO]: MagicCobblestoneGenerator 2.4.0-SNAPSHOT-b228 (ENABLED)
[10:04:43 INFO]: RoseStackerHook 1.0.0-SNAPSHOT-b3 (ENABLED)
[10:04:43 INFO]: TwerkingForTrees 1.4.3 (ENABLED)
[10:04:43 INFO]: Upgrades 0.2.0-SNAPSHOT-b84 (ENABLED)
[10:04:43 INFO]: Visit 1.3.0 (ENABLED)
[10:04:43 INFO]: Warps 1.10.2 (ENABLED)

**Comment by tastybento** [comment]
(From @iRedFox report on Discord)

Console error screenshot: 
![image](https://user-images.githubusercontent.com/4407265/130084650-3b84db4f-6a98-48b4-bcb1-3c426eb6da65.png)

Screenshot of usage:
![image](https://user-images.githubusercontent.com/4407265/130084876-0d32a86c-69f9-4f04-820c-41e9b70742eb.png)

Output if `/island info`:
![image](https://user-images.githubusercontent.com/4407265/130084947-a81a9a57-16a1-42ad-8ad0-b599cf1adcdb.png)

Shot of island:
![image](https://user-images.githubusercontent.com/4407265/130084612-47facbe8-3b37-4a9d-9950-9e77fc022ba0.png)

**Comment by tastybento** [comment]
@iRedFox If you remove RoseStacker and run `/island info`, does it return with a calculation?

**Comment by iRedFox** [comment]
Nope, doesn't return with calculation

**Comment by junxiesky** [comment]
Yes, this problem exists. I have encountered this problem too. When can I fix it?

**Comment by junxiesky** [comment]
Yes, this problem exists. I have encountered this problem too. When can you fix it?

**Comment by junxiesky** [comment]
Yes, this problem exists. I have encountered this problem too. When can you fix it?

**Comment by junxiesky** [comment]
> > 是的，这个问题存在。我也遇到过这个问题。什么时候能修好？
> 
> 嗯，根据提供的当前信息无法修复。如果您看到此问题，那么我需要一份您的世界的副本，以及压缩并共享的 BentoBox 文件夹，以及出现问题的岛屿位置。此外，如果发生时控制台有任何日志。一旦我有了这些，我将尝试复制并查看它是否发生，我可以捕获正在发生的内部错误。另外，分享您的输出，`bbox version`以便我知道您正在使用的特定版本号。

okay,When I encounter this problem next time I will provide these to you.
